manager/orchestrator/constraintenforcer/constraint_enforcer.go

```diff
@@ -83,10 +84,15 @@ func (ce *ConstraintEnforcer) rejectNoncompliantTasks(node *api.Node) {
 		log.L.WithError(err).Errorf("failed to list tasks for node ID %s", node.ID)
 	}
 
-	var availableMemoryBytes, availableNanoCPUs int64
+	available := &api.Resources{}
+	fakeStore := make([]*api.GenericResource, 0)
```

Comment:
This can be `var fakeStore []*api.GenericResource` (even though you're passing its address to a function).

What does the name `fakeStore` mean?